ani-cli

```diff
@@ -408,14 +454,21 @@ case "$search" in
             [ -z "$query" ] && exit 1
         fi
         query=$(printf "%s" "$query" | sed "s| |+|g")
-        anime_list=$(search_anime "$query")
-        [ -z "$anime_list" ] && die "No results found!"
-        [ "$index" -eq "$index" ] 2>/dev/null && result=$(printf "%s" "$anime_list" | sed -n "${index}p")
-        [ -z "$index" ] && result=$(printf "%s" "$anime_list" | nl -w 2 | sed 's/^[[:space:]]//' | nth "Select anime: ")
-        [ -z "$result" ] && exit 1
-        title=$(printf "%s" "$result" | cut -f2)
+        if [ "$thumbnails" = "1" ]; then
+            search_anime "$query"
+            image_preview_fzf
+            title=$(printf "%s" "$choice" | sed -nE "s/  .* \([0-9]* episodes\)  (.*)/\1/p")
+            id=$(printf "%s" "$choice" | sed -nE "s/  .* \([0-9]* episodes\)  (.*)\.jpg/\1/p")
+        else
+            anime_list=$(search_anime "$query")
+            [ -z "$anime_list" ] && die "No results found!"
+            [ "$index" -eq "$index" ] 2>/dev/null && result=$(printf "%s" "$anime_list" | sed -n "${index}p")
+            [ -z "$index" ] && result=$(printf "%s" "$anime_list" | nl -w 2 | sed 's/^[[:space:]]//' | nth "Select anime: ")
+            [ -z "$result" ] && exit 1
+            title=$(printf "%s" "$result" | cut -f2)
+            id=$(printf "%s" "$result" | cut -f1)
+        fi
```

Comment:
The placement of this block does not run when we have history search.
Additionally, I see no reason for a true if-else.
The thumbnails feature is an addition. 
The prior behaviour should not be in an else block.

Reply:
someone do this, im lazy